apps/payment/tests/api_tests.py

```diff
@@ -212,7 +212,7 @@ def test_event_payment_fails_with_fake_payment_method_id(self):
 
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertIn(
-            f"No such PaymentMethod: {fake_payment_method_id}", response.json()[0]
+            f"No such PaymentMethod: '{fake_payment_method_id}'", response.json()[0]
```

Comment:
Are these actually supposed to be strings, they seem like they should be the variable?

Reply:
They are f-strings right now, with `fake_payment_method_id` as a variable? The problem is that Stripe apparently changed their API, so that the ID is quoted in `''` now, see https://github.com/dotkom/onlineweb4/runs/954000945

Reply:
Ah, well yes. That totally doesn't make things more confusing stripe.